cleaned up the chunk 2

mir-abir-hossain opened the pull request:
There are lot of English dialogues. Kept them as it is.
Fixed spelling mistakes according to the original book.
Some lines were not extracted. Added them.

mir-abir-hossain committed:
cleaned up the chunk 2

mir-abir-hossain commented:
@dg1223 please do a review when you have time.

dg1223 reviewed (changes requested):
Please fix the following spellings. I haven't provided the replacements because I think you'd understand all of them anyway. The numbers denote line numbers when you open the committed file on github or a text editor such as notepad++, sublime text etc. I have not been able to check if all lines are present in the extracted text. I trust your diligence on it.

18: বজ্তৃতা
21: উর্দ্দূতে
29: দক্ষিনপদ্থী
51: সায়ত্্
84: ইনফ্যানট্র
132: হস্তাস্তরও
138: শেষঅন্দি
167: শ্বেতস্ত্রাসের
172: ফায়দ৷
187: উদ্বিম্ন
194: ভুষ্ট্রোকে
269: ক্রুমাম্বয়ে
301: সারবস্ত
472: ওরা
541: আত্তরিক
565: বিক্ষোরম্মুখ
578: মারশলি
599: মার্শল'ল
636: গর্নিত
646: দ্
700: তীব্‌
726: ইশ্্‌তেহার
753: সশুক্রবার
760: hel
771: খল্পর
805: কর্সছিলেন
813: নির্বিধায়
817: মাশলি'ল
855: মন্ত্রীত্ববের
857: যুক্তস্ন্ট
865: সোহরাওয়াদ্দী
869: অর্্তভুক্ত
876: কযমু্যুনিষ্ট
904: দ্বি-বন্ডিত
911: মস্কোপছ্থী
927: ফ্ষূর্লা
928: বক্তবাটি
929: প্রানিধানযোগ্য
996: তুষ্টো
1020: জাতিসম্ত্বার
1036: উননয়নের
1074: ইস্পোর্ট
1122: উনুনয়ন, য।
1127: মর্মি
1161: ক্রান্তিলয্নে
1212: এখম
1217: দিতীয় গরে
1251: সব্বেপিরি
1258: গ্রুব
1262: চাষ্ষুস
1263: জযল্লাদ
1269: আস্তরিকায়ও
1271: ;
1346: বাস্ধব
1347: সেন!
1380: ইমারজেপী
1394: শুভাকাজ্খী
1442: আবাস্তব
1507: তারতীয়
1570: অঙ্ধকারে
1614: হ্া
1659: নিধধারিত
1666: থাতির
1688: পরিস্থৃতিতেই
1715: লঙ্জা
1777: ফিল্াগান
1840: লিঙ্ত
1856: ক্ষমতালিন্মু
1869: জিষ্জিরে
1887: ব্রিপাক্ষীয়
1890: সর্বোপিরিসরে
1891: বেঙঈমানীর
2016: অ্রফিসে
2025: চম্কে
2041: সংঘরাম
2067: server -> served, ন্যাপ্কিনে
2084: ওরোথ্‌ -> ওরোখ
2091: courrage -> courage
2131: ডিতু
2133: হুল্পরবাজ

mir-abir-hossain committed:
fixed additional mistake

mir-abir-hossain committed:
added missing lines

mir-abir-hossain commented:
Thank you very much @dg1223 for pointing out the errors. I have fixed those spelling mistakes and some additional more spelling mistakes. Also I was able to find out almost 8/9 missing lines that I didn't capture in the first try. I have added all the lines and I have made sure no more line is missing. About spelling mistakes, I am hopeful 95%+ mistakes are corrected. 
Thanks for the wonderful review. Please if you have time can you go through it once and then merge it?

dg1223 reviewed (changes requested):
Great work! I think this extract was one of the most challenging. Only these ones are remaining:

185: উদ্বিঘ্ন -> উদ্বিগ্ন
566: মারশলি'ল -> মার্শাল'ল
581, 599: মার্শল'ল -> মার্শাল'ল
646: দ্ (remove this)
753: সশুক্রবার -> শুক্রবার
813: নির্বিধায় -> নির্দ্বিধায়
1123: গণঅভ্যুথানের -> গণঅভ্যুত্থানের
1269: আস্তরিকায়ও -> আন্তরিকতায়ও (incorrect in the book)
1858: একনায়কতত্ব -> একনায়কত্ব
2025: চম্কে -> চমকে
2067: ন্যাপ্কিনে -> ন্যাপকিনে

mir-abir-hossain committed:
refixing

mir-abir-hossain commented:
Thank you for the rereview। Done the corrections. @dg1223

dg1223 reviewed (approved):
Looks good, thanks!

Closing issue #13

dg1223 commented:
@mir-abir-hossain Now that your PR has been merged, maybe you can delete [your branch](https://github.com/mir-abir-hossain/real-history-of-Bangladesh/tree/data_mir_abir_hossain_chunk_2_cleanup) from here:

https://github.com/mir-abir-hossain/real-history-of-Bangladesh/branches
